src/components/OSCALControlModification.js

```diff
@@ -19,11 +19,15 @@ const OSCALControlModificationsButton = styled(IconButton)(
  * Create a typography html object for addsOrRemovesLabel
  *
  * @param {Object} addsElements add or remove object to map into html
- * @param {String} addsLabel boolean variable, true if adding
+ * @param {String} addsRemovesLabel string variable
  * @param {String} controlPartId Control part ID to match
  * @returns html object
  */
-function getAlterAddsOrRemovesDisplay(addsElements, addsLabel, controlPartId) {
+function getAlterAddsOrRemovesDisplay(
+  addsElements,
```

Comment:
Is this only the `adds` elements? 

I think you could change a lot of the `addsorremoves` to be `alters`. So `altersElements`, `altersLabel`, etc.